src/views/PurchaseOrderDetail.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button default-href="/" slot="start"></ion-back-button>
+        <ion-title>{{$t("Purchase Order Details")}}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button @click="addProduct"><ion-icon :icon="add"/></ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <div>
+        <ion-item lines="none">
+          <h1>{{$t("Purchase Order")}}: PO10291</h1>
+        </ion-item>
+        <ion-item>
+          {{$t("Scan Items")}}
+          <ion-label>
+            <ion-input placeholder="Scan barcodes to receive"></ion-input>
+          </ion-label>
+        </ion-item>
+        <ion-card>
+          <div class="product-info">
+            <ion-item lines="none">
+              <ion-thumbnail slot="start">
+                <img src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+              </ion-thumbnail>
+              <ion-label>
+                <h2>Chaz Kangeroo Hoodie-XS-Green</h2>
+                <p>12203</p>
+              </ion-label>
+            </ion-item>
+            <ion-item class="product-count">
+              <ion-input type="number" value="0" min="0"></ion-input>
+            </ion-item>
+          </div>
+          <ion-item class="border-top">
+            <ion-button color="primary" slot="start" fill="outline">
```

Comment:
Why color primary is given here ? Default color is already primary.

Reply:
Removed Ma'am.